### ISSUE TITLE
Looc not working on fresh typescript create-react-app 

### ISSUE THREAD

**Issue by aadamsx** [jlkiri/looc#7]
On Windows 10

> npx create-react-app test-looc --template typescript
> cd test-looc
> npm install looc --save-dev

```
// Created IDCards.tsx (based off your example to ./src)

import React from 'react';

interface IDCardProps {
  firstName: string;
  lastName: string;
  title: string;
  id: number;
  picShape: "squared" | "round";
  telephone: string;
}

const IDCard: React.FC<IDCardProps> = ({
  firstName = "",
  lastName = "",
  title = "",
  id = 0,
  telephone = "",
  picShape = "round",
}) => {
  /* your components code */
};

export default IDCard;
```

> npx looc start src/IDCard.tsx

```
// The first time

Installing libraries:  [ 'react', 'react-dom', '@material-ui/core' ]
LOOC ERROR: Error: Command failed with exit code 1: npx snowpack --config C:\Users\aaadams\Projects\test-looc\.cache\snowpack.config.json --source pika
- snowpack installing...
warn(@material-ui/core): Not found in "dependencies". Using latest package version...
⠼ snowpack installing...
× connect ETIMEDOUT 35.202.100.12:443
ℹ "source: pika" mode enabled. Behavior is still experimental and may change before the next major version...
    at makeError (C:\Users\aaadams\Projects\test-looc\node_modules\looc\node_modules\execa\lib\error.js:58:11)
    at handlePromise (C:\Users\aaadams\Projects\test-looc\node_modules\looc\node_modules\execa\index.js:114:26)
    at processTicksAndRejections (internal/process/task_queues.js:89:5) {
  shortMessage: 'Command failed with exit code 1: npx snowpack --config ' +
    'C:\\Users\\aaadams\\Projects\\test-looc\\.cache\\snowpack.config.json ' +
    '--source pika',
  command: 'npx snowpack --config ' +
    'C:\\Users\\aaadams\\Projects\\test-looc\\.cache\\snowpack.config.json ' +
    '--source pika',
  exitCode: 1,
  signal: undefined,
  signalDescription: undefined,
  stdout: 'ℹ "source: pika" mode enabled. Behavior is still ' +
    'experimental and may change before the next major ' +
    'version...',
  stderr: '- snowpack installing... \nwarn(@material-ui/core): Not ' +
    'found in "dependencies". Using latest package version...\n⠼ ' +
    'snowpack installing... \n× connect ETIMEDOUT ' +
    '35.202.100.12:443',
  failed: true,
  timedOut: false,
  isCanceled: false,
  killed: false
}
Install finished!

// Just hung forever, had to terminate here
Terminate batch job (Y/N)?

```

> npx looc start src/IDCard.tsx

```
// Now running again it just stays at install finished
Skipping previously installed libraries:  [ 'react', 'react-dom', '@material-ui/core' ]
Install finished!

```

While it's sitting at Install finished, I try to go to localhost:3000 but get a "this site can't be reached"

**Comment by jlkiri** [comment]
This is indeed a bug. 

Apparently `snowpack`  could not install libraries because of network problems but it seems to think they're installed, because it outputs a json with a list of installed libs even before it installs them. Should not take much time to fix.

**Comment by jlkiri** [comment]
@aadamsx Fixed in `0.6.2`!
